Assets/db/dane-testowe.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO "main"."SCORES" ("score", "nickname", "data") VALUES ('52', 'gracz7', '2020-01-25 09:15:32.311');
```

Comment:
Perhaps the name of this file should be in english? And let's follow `UpperCamelCase` naming convention.

Reply:
Name of file change - 'TestData'